plugins/generator-1.18.2/forge-1.18.2/templates/modbase/accesstransformer.cfg.ftl

```diff
@@ -23,4 +23,8 @@ public net.minecraft.world.level.GameRules$IntegerValue m_46312_(I)Lnet/minecraf
 
 <#if w.hasGameRulesOfType("Logic")>
 public net.minecraft.world.level.GameRules$BooleanValue m_46250_(Z)Lnet/minecraft/world/level/GameRules$Type; #create
+</#if>
+
+<#if w.hasElementsOfType("villagerprofession")>
+public net.minecraft.world.entity.ai.village.poi.PoiType m_27367_(Lnet/minecraft/world/entity/ai/village/poi/PoiType;)Lnet/minecraft/world/entity/ai/village/poi/PoiType; # registerBlockStates

```

Comment:
Where is this used?

Reply:
Huh. When was this added? It wasn't there last time I forked.

Reply:
Apr 25, 2022 in commit https://github.com/MCreator/MCreator/commit/a401b83dd749e2902f671a7fbfddbbc59bda7f80

Reply:
Oh, it's only on 1.18.2. I don't think this is needed anywhere.